EIPS/eip-6110.md

```diff
@@ -76,11 +76,26 @@ Beginning with the `FORK_BLOCK`, each deposit accumulated in the block **MUST**
 in the order they appear in the logs. To illustrate:
 
 ```python
-def parse_deposit_data(deposit_event_data) -> bytes[]:
-  """
-  Parses deposit data from DepositContract.DepositEvent data
-  """
-  pass
+def parse_deposit_data(deposit_event_data):
+    """
+    Parses deposit data from DepositContract.DepositEvent data.
+    """
+
+    # TODO: verify byte length of deposit_event_data? Should be 576. Should this be part of the spec? 
```

Comment:
I think it has to be. The simplest is to require the length of the log data to be exactly this (otherwise, no request is noted).

Reply:
Whoops forgot to remove this TODO. It is currently at line 115 `is_deposit_event = (len(log.topics) > 0 and log.topics[0] == DEPOSIT_EVENT_SIGNATURE_HASH and len(log.data) === 576)`. I'll remove the TODO :)